components/script/dom/imagebitmap.rs

```diff
@@ -39,6 +42,14 @@ impl ImageBitmap {
 
         Ok(reflect_dom_object(imagebitmap, global))
     }
+
+    pub fn set_bitmap_data(&self, data: DomRefCell<Vec<u8>>) {
+        *self.bitmap_data.borrow_mut() = data.borrow_mut().to_vec();
```

Comment:
This should take a `Vec<u8>` argument instead, and use `*self.bitmap_data.borrow_mut() = data;`

Reply:
Made this change and on the calling function as well.